src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://us-cdbr-east-02.cleardb.com/heroku_bf61bac08e7d5ce
+spring.datasource.username=be954c4707bc32
+spring.datasource.password=ENC(epj2Y3gphq6164k15apQphtgq8aj12qZ)
+spring.jpa.open-in-view=true
+spring.datasource.hikari.connectionTimeout=45000 
```

Comment:
Could you explain what these values are and what they do? Nice work on figuring that out btw :)

Reply:
I added comments to the `application.properties` file to explain each of these values.

The first four lines (copied below) set up the database connection.
```
spring.jpa.hibernate.ddl-auto=update
spring.datasource.url=jdbc:mysql://us-cdbr-east-02.cleardb.com/heroku_bf61bac08e7d5ce
spring.datasource.username=be954c4707bc32
spring.datasource.password=ENC(epj2Y3gphq6164k15apQphtgq8aj12qZ)
```
The next line (copied below) enables Open Session in View. You can see more details in this [StackOverflow post](https://stackoverflow.com/questions/30549489/what-is-this-spring-jpa-open-in-view-true-property-in-spring-boot); OSIV is enabled by default, and a warning is provided when a value is not set. The GitHub response actually says to set this to `false`, but I think extra work needs to be done if that's the case, and since `true` is the default, I wanted to set it to that at first. See more info: [[1]](https://www.yawintutor.com/warn-3840-spring-jpa-open-in-view-is-enabled-default/).
```
spring.jpa.open-in-view=true
```
The final lines (copied below) set configuration parameters for HikariCP; read more [here](https://www.baeldung.com/spring-boot-hikari#hikari-configuration). The `maxLifetime` value has to be less than `idleTimeout` (and, if I understand correctly, `connectionTimeout` as well) as explained in this [StackOverflow post](https://stackoverflow.com/questions/46830159/hikari-failed-to-validate-connection-because-connection-is-closed).
```
spring.datasource.hikari.connectionTimeout=45000 
spring.datasource.hikari.idleTimeout=600000 
spring.datasource.hikari.maxLifetime=30000
```